docs/riff_streaming_inmemory-gateway.md

```diff
@@ -8,7 +8,10 @@ title: "riff streaming inmemory-gateway"
 
 ### Synopsis
 
-<todo>
+The in-memory Gateway encapsulates the address of a streaming gateway and a no-op Provisioner instance.
```

Comment:
Too much implementation detail IMHO. Why not just say that the in-memory gateway "stores messages in memory"?